build.gradle

```diff
@@ -21,8 +21,8 @@ group = 'com.thoughtworks.gocd'
 
 gocdPlugin {
   id = 'com.thoughtworks.gocd.elastic-agent.ecs'
-  pluginVersion = '7.0.0'
-  goCdVersion = '19.3.0'
+  pluginVersion = '7.1.0'
+  goCdVersion = '20.9.0'
```

Comment:
@kritika-singh3 why do we need to bump up GoCD version?

Reply:
We are compiling with Java 15 now - the support of the same is introduced in 20.9.0 - it won't work for previous releases.

Reply:
Ok, thanks.